### ISSUE TITLE
Make touch targets larger for accessibility

### ISSUE THREAD

**Issue by albullington** [inaturalist/SeekReactNative#160]
Google Play outlines a few potential accessibility improvements. The following elements should have a larger touchable area:

- [x] Back button -> GreenHeader, ChallengeDetail
- [x] Back button -> Camera, SpeciesDetail
- [x] Camera help button
- [x] Menu icon and bird/notifications icon -> Footer
- [x] Location picker and challenges text -> Home
- [x] Start now buttons -> ChallengeScreen
- [x] CC button on SpeciesPhotos
- [x] Checkbox -> LicensePhotos
- [ ] Date picker -> AgeVerify

These require design changes:
- [ ] Green button -> Location Picker, Getting Started, ChallengeDetail, Login screens
- [ ] Forgot your password text
- [ ] InputFields -> Login and signup screens, Location Picker
- [ ] Native / Threatened Status buttons, SpeciesDetail
- [ ] Camera / Gallery Tabs
